examples/lambda-url/src/server.ts

```
@@ -0,0 +1,80 @@
+    }
+
+    type StreamifyHandler<TEvent = any, TResult = any> = (
+      event: TEvent,
+      responseStream: awslambda.HttpResponseStream,
+      context: Context,
+    ) => TResult | Promise<TResult>;
+
+    function streamifyResponse<TEvent = any, TResult = void>(
+      handler: StreamifyHandler<TEvent, TResult>,
+    ): StreamifyHandler<TEvent, TResult>;
+  }
```

ESLint

[error] ES2015 module syntax is preferred over namespaces.

(@typescript-eslint/no-namespace)